### ISSUE TITLE
Grid inside flexbox broken in Safari

### ISSUE THREAD

**Issue by pareeohnos** [ag-grid/ag-grid#4132]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

We've had a report from a customer that one of our tables is completely blank when they load it. Upon investigation, the problem appears to only occur when using Safari, and the table is inside a flexbox. When I twas developed, we followed the ag-grid guide on putting the grid in a flexbox and it's worked just fine, apart from in Safari. 

To see an example of the issue, you can view the main ag-grid website where the problem exists, so I know it's not unique to our application. For example the [documentation about column pinning](https://www.ag-grid.com/javascript-grid-pinning/) shows blank tables in Safari.

**Current behaviour**

Tables in a flexbox are completely blank in Safari

**Expected behaviour**

Tables to render correctly

**Please tell us about your environment:**
OS: OS X 10.15.7
Browser: Safari 14

* **ag-Grid version:** X.X.X
24.0.0

* **Browser:**
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0 Safari/605.1.15

**Comment by makinggoodsoftware** [comment]
Hi,

We just test this in Safari, it all seems to work fin in our end

Can you provide a reproducible scenario using one of our examples or your source code in a plunker and we will investigate?

Accepted reproducible scenarios are
- A description of the detailed steps to reproduce your behaviour in one of our examples in the docs.
- A plunker

If you decide to send us a plunkr, from any example in our website use the plunkr button in there to fork your own code by following the steps below:

- Select the framework that is appropriate to you from the drop-down
- Open it in plunker. (Use the button plunker in our example)
- Add your changes so that the behaviour is reproduced
- Save and Freeze the plunker(On the top left corner)
- Send us the link to the plunker(You can copy the URL from the browser)

If reporting a bug make sure to state.

Actual behaviour.
Expected behaviour. If possible back this up with our docs/examples if possible

Note that this issue is getting closed, but you are welcomed to create a new issue if you can provide us with a reproducible scenario

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.

**Comment by pareeohnos** [comment]
@makinggoodsoftware since I opened this bug, it seems a new version was released (24.1.0) which appears to have fixed the issue. I can no longer replicate it on the ag-grid website. I'll try it out in our codebase and if it's still broken there, I'll reopen and try and reproduce
